mgl-pax.asd

```diff
@@ -62,6 +62,7 @@
                              (:file "locatives")
                              (:file "sections")
                              (:file "interaction-mode")
+                             (:file "sly" :if-feature :slynk)
```

Comment:
Is the case where slynk is loaded afterwards worth thinking about?

Reply:
It might be (say you have an existing Lisp image with MGL-PAX, where you decided to load Slynk and connect into from Emacs after the fact), we'd just need to think of a place where it would be appropriate to load that. Perhaps we could maintain a mapping between backends and support systems, and have `CALL-WITH-LISP-INTERACTION-MODE` loading that as appropriate, although I am inclined to dig into ASDF to see if we could tell it to do that for us (i.e. load this third system whenever those other ones happen to be loaded, whichever the order).

Reply:
There's already autoloading on the CL side. Perhaps this could be another.

Reply:
I've generalized the autoloading logic to be callable from contexts other than autoloaded function definitions, and had that autoload the rest of the Sly implementation on the body of Sly's `call-with-lisp-interaction-mode`.